src/MICore/CommandFactories/lldb.cs

```diff
@@ -160,5 +160,12 @@ public override Task Catch(string name, bool onlyOnce = false, ResultClass resul
         {
             throw new NotImplementedException("lldb catch command");
         }
+
+        public async override Task<string> VarAssign(string variableName, string expression, int threadId, uint frameLevel)
+        {
+            Results results = await VarCreate(expression, threadId, frameLevel, 0, ResultClass.done);
+            string value = results.FindString("value");
```

Comment:
Does `value` have any extra stuff on it that LLDB wouldn't want on an assignment? Ex: if you have a `char*` variable and assign it `(char*)0x12345` (where 12345 is the address of a valid string) will this wind up adding anything extra? Might be interesting to indicate what types we have tested.